language/stdlib/modules/CoreAddresses.move

```diff
@@ -34,6 +34,13 @@ module CoreAddresses {
         0x0
     }
 
+    /// The reserved address where all core modules are published. When module
```

Comment:
Maybe remove comment about "when module publishing is closed" to avoid confusion? We can update it if publishing opens up.